test/integration/dialects/mssql/regressions.test.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const chai = require('chai');
+const _ = require('lodash');
```

Comment:
prefer importing `lodash/times` instead of `lodash`, lodash is big and can have an noticeable impact on load times if it's fully loaded VS loading a single function